pixl_export/src/pixl_export/main.py

```diff
@@ -0,0 +1,110 @@
+#  Copyright (c) University College London Hospitals NHS Foundation Trust
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+"""pixl_export module is an EHR extraction service app"""
+
+from __future__ import annotations
+
+import importlib.metadata
+import logging
+from datetime import (
+    datetime,  # noqa: TCH003, always import datetime otherwise pydantic throws error
+)
+from pathlib import Path
+
+from core.exports import ParquetExport
+from core.project_config import load_project_config
+from core.rest_api.router import router
+from core.uploader import get_uploader
```

Comment:
Since `core.uploader` is now only used by `pixl_export` should we just move it over from `core`?

Reply:
I tried this but it's complicated by `ParquetExport` (currently in core) using parts of `core.uploader`. We could move `ParquetExport` to export-api as well, but then we run into the fact that `ParquetExport` is used by both export-api and CLI.
I tried moving various bits around and got into a circular import hell. I am probably missing some obvious solution, but it could be that the fact we have CLI doing exports makes this inherently harder than it needs to be. Maybe worth a separate issue/PR?

Reply:
Now issue #384